volume/drivers/adapter.go

```diff
@@ -124,13 +126,15 @@ type volumeAdapter struct {
 	name         string
 	baseHostPath string
 	driverName   string
-	eMount       string // ephemeral host volume path
+	eMount       string    // ephemeral host volume path
+	createdAt    time.Time //time the directory was created
```

Comment:
nit: try to be consistent with the comments (this line doesn't have a leading space after `//`